core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/identity/IdentityPlugin.java

```diff
@@ -49,11 +77,16 @@ public void addConfigurations(ConfigurationRegistry registry) {
 	@Override
 	public void init(SnowOwlConfiguration configuration, Environment env) throws Exception {
 		final IdentityConfiguration conf = configuration.getModuleConfig(IdentityConfiguration.class);
-		final List<IdentityProvider> providers = createProviders(env, conf.getProviderConfigurations());
+		final List<IdentityProvider> providers = createProviders(env, conf.getProviderConfigurations() == null ? List.of() : conf.getProviderConfigurations());
 		
-		IdentityProvider identityProvider = null; 
+		IdentityProvider identityProvider = null;
 		if (providers.isEmpty()) {
-			identityProvider = IdentityProvider.NOOP;
+			// if there are no providers, but the issuer is external and there is a JWS configured, then assume we are using access tokens and Snow Owl is a resource server
+			if (conf.isExternalIssuer() || !Strings.isNullOrEmpty(conf.getJws())) {
```

Comment:
Is it allowed/useful to set a non-Snow Owl issuer in the config, but leave the JWS property empty? We obtain a `DecodedJWT` via verification in `IdentityProvider`, and I'd think it will break when it tries to use the algorithm implementation that always throws `BadRequestException`s.

Reply:
If a non-Snow Owl issuer is configured, then a JWS must be specified, otherwise, it will throw the disabled BadRequest exception message, which is the indication of the misconfiguration (I did not want to use an HTTP 50x error).
I can add another check that the JWS parameter is required when non-Snow Owl issuer is configured. What do you think?

Reply:
Would it work if this check was reduced to:

```suggestion
			if (!Strings.isNullOrEmpty(conf.getJws())) {
```

Reply:
In this case, Snow Owl would need to generate a JWT, but that would require an identity provider to fetch the user data from plus verify the user/pass. So, no unfortunately this is not an option here.